client/components/section-nav/index.jsx

```diff
@@ -152,6 +154,13 @@ class SectionNav extends Component {
 		};
 	}
 
+	generateOnSearchOpen( existingOnSearchOpen ) {
+		return ( ...args ) => {
+			existingOnSearchOpen( ...args );
+			this.closeMobilePanel();
```

Comment:
Closes the mobile panel when running the enhanced version of `onSearchOpen`. This avoids having a search box that is the same height of the dropdown.